manifests/wego-app/helm_watcher_role.yaml.tpl

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: helm-watcher-role
+  namespace: {{ .Namespace }}
+rules:
+  - apiGroups:
+      - source.toolkit.fluxcd.io
+    resources:
+      - helmrepositories
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - source.toolkit.fluxcd.io
+    resources:
+      - helmrepositories/finalizers
+    verbs:
+      - create
+      - delete
+      - get
+      - patch
+      - update
+  - apiGroups:
+      - source.toolkit.fluxcd.io
+    resources:
+      - helmrepositories/status
+    verbs:
+      - get
```

Comment:
is this covered by the below?
```
  - apiGroups:
      - source.toolkit.fluxcd.io
    resources:
      - helmrepositories
    verbs:
      - get
      - list
      - watch
```

Reply:
like `helmrepositories/status` resource is a sub resource of `helmrepositories`

Reply:
As far as I remember it doesn't propagate. Subresources are used with the exact API. If you don't specify them, you don't have the same access. I mean, you can use something like `*` to grant access to all resources, but I don't want that. I only need get and only for specific resources.

Reply:
:ok_hand: makes sense